description_cocina_schema_v4.json

```diff
@@ -0,0 +1,429 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "http://cocina.sul.stanford.edu/schemas/Description",
+    "title": "Description",
+    "description": "Descriptive metadata for a digital object.",
+    "type": "object",
+    "definitions": {
+        "AppliesTo": {
+            "description": "Property model for indicating the parts, aspects, or versions of the resource to which a descriptive element is applicable.",
+            "type": "object",
+            "properties": {
+                "appliesTo": {
+                    "type": "array",
+                    "items": {"$ref": "#/definitions/DescriptiveBasicValue"}
+                }
+            }
+        },
+        "Contributor": {
+            "description": "Property model for describing agents contributing in some way to the creation and history of the resource.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Names associated with a contributor.",
+                    "type": "array",
+                    "items": {"$ref": "#/definitions/DescriptiveValue"}
+                },
+                "type": {
+                    "description": "Entity type of the contributor (person, organization, etc.).",
+                    "type": "string"
```

Comment:
I've tried to stay away from enums as I don't want to be overly prescriptive at this point. I'm leaning towards seeing that level of detail as belonging to a metadata application profile rather than the data model, but that could change as things progress.

Reply:
OK